README.md

```diff
@@ -21,20 +21,20 @@ Version of this bioimage.io configuration specification. This is mandatory, and
 The recommended behavior for the implementation is to keep backward compatibility, and throw error if the model yaml is in an unsupported format version.
 
 - `name`
-Name of the specification. This name should equal the name of any existing, logically equivalent object of the same category in another language/framework.
+Name of this specification.
```

Comment:
Should this be "Name of this model"?

And perhaps we should define the naming rule. E.g. this should be a human readable name, it should only contain `a-z`, `A-Z`, `-`, `0-9` or space, no more than 36 letters.

Reply:
Agreed to all, I will update it.